terraform/environments/preview/s3_buckets.tf

```diff
@@ -306,9 +381,57 @@ resource "aws_s3_bucket" "g7-draft-documents_bucket" {
   }
 }
 
+resource "aws_s3_bucket_policy" "g7-draft-documents_bucket" {
+  bucket = "${aws_s3_bucket.g7-draft-documents_bucket.id}"
+
+  policy = <<POLICY
```

Comment:
Is there a reason we're using an inline policy here over a `policy_document`? I think we want to use the latter where possible.

Reply:
I was following the convention of existing policies, if they were inlined I left them inlined. But I can change them, if everyone prefers the TF syntax over inlined JSON.